src/row.jl

```diff
@@ -252,15 +252,17 @@ ArrowTypes.arrowname(::Type{<:Curve}) = CURVE_ARROW_NAME
 ArrowTypes.JuliaType(::Val{CURVE_ARROW_NAME}) = Curve
 
 """
-    const ClassRow = Legolas.@row("lighthouse.class@1",
-                                  class_index::Union{Int64,Symbol}
+    const ClassRow = Legolas.@row("lighthouse.class@2",
+                                   class_labels::Union{Missing,Vector{String}},
+                                   class_index::Union{Int64,Symbol})
 
-A type alias for [`Legolas.Row{typeof(Legolas.Schema("lighthouse.class@1"))}`](https://beacon-biosignals.github.io/Legolas.jl/stable/#Legolas.@row)
+A type alias for [`Legolas.Row{typeof(Legolas.Schema("lighthouse.class@2"))}`](https://beacon-biosignals.github.io/Legolas.jl/stable/#Legolas.@row)
 representing a single column `class_index` that holds either an integer or the value
-`:multiclass`.
+`:multiclass`, and the associated class names
 """
-const ClassRow = Legolas.@row("lighthouse.class@1",
-                              class_index::Union{Int64,Symbol} = check_valid_class(class_index))
+const ClassRow = Legolas.@row("lighthouse.class@2",
```

Comment:
These don't need to be `@2`; the new schema is compatible with the old one, since the new fields are allowed to be missing. That means if you try to load a table saved with the old schema, it will still work (if we don't bump the version). The official rule from https://beacon-biosignals.github.io/Legolas.jl/dev/schema/#Simple-Integer-Versioning:-You-Break-It,-You-Bump-It-1 is 

> If an update is made to a schema that potentially requires existing data to be rewritten in order to comply with the updated schema, then the version integer associated with that schema should be incremented.

Therefore we don't have to bump the version! Which is good, because bumping the version is a big pain; it will make all the old tables unloadable from this version of Lighthouse unless keep put the code for both schemas.

Reply:
perfect! I will do a quick un bump here then!!